.github/workflows/index-render-template.yaml

```diff
@@ -48,40 +45,41 @@ jobs:
     steps:
     - name: Checkout the repository
       uses: actions/checkout@v4
+      with:
+        ref: release-v1.18.x
+    - name: Configure Docker authentication
+      env:
+        DOCKER_CONFIG: /home/runner/.docker
+        DOCKER_JSON: ${{ secrets.DOCKER_REGISTRY_SECRET }}
+      run: |
+          echo ${DOCKER_JSON} > $DOCKER_CONFIG/config.json
     - name: Generate ${{matrix.version}} catalog
       run: |
-        echo ${{matrix.version}}
         curl -sSfLo /usr/local/bin/opm "https://github.com/operator-framework/operator-registry/releases/download/v1.47.0/linux-amd64-opm"
         chmod +x /usr/local/bin/opm
+        ./hack/index-render-template.sh ${{matrix.version}} ${{github.event.inputs.environment}}
```

Comment:
which index-render-template.sh will be called ?? asking because i don't see changes for main and next